mkt/websites/indexers.py

```diff
@@ -29,7 +29,7 @@ def get_mapping(cls):
                 'properties': {
                     'id': {'type': 'long'},
                     'category': cls.string_not_analyzed(),
-                    'created': {'type': 'date', 'format': 'dateOptionalTime'},
+                    'created': {'format': 'dateOptionalTime'},
```

Comment:
Yeah, it was added on purpose to reproduce the issue that caused us to revert this patch the previous time. If you look at the travis build, you'll see a failure caused by this particular change, which is fine, and then a bunch of other failures that should not happen. The goal is to find out what causes those errors and then when we do, we can remove this part of the patch.